src/lib/backendClient.ts

```
@@ -297,9 +257,7 @@ class BackendApiClient {
   }
 
   // Token management
-  async getToken(): Promise<string | null> {
-    // Race condition: ensure token is loaded before returning
-    await this.ensureTokenLoaded();
+  getToken(): string | null {
     return this.token;
   }
```

Biome (2.1.2)

[error] Duplicate class member name "getToken"



(lint/suspicious/noDuplicateClassMembers)